example.py

```diff
@@ -37,8 +37,10 @@ def main():
     print()
     time.sleep(1)
```

Comment:
nit: make this 1 a constant

Reply:
while we are at it can we wrap this `print()` in a helper method

Reply:
Let's add that as a task to Jira.